packages/css-library/src/stylesheets/_override-function.scss

```diff
@@ -1,10 +1,11 @@
-@use 'sass:math';
+// Removing sass math for now because it doesn't play nicely with vets-website
+// @use 'sass:math'; 
 
 $uswds-base: 16px;                    // USWDS base
 $formation-base: 10px;                // Formation base
 
-$f2u-coeff: math.div($formation-base, $uswds-base); // Formation  to USDWS conversion coeffient
-$u2f-coeff: math.div($uswds-base, $formation-base); // USDWS to Formation conversion coeffient
+$f2u-coeff: $formation-base / $uswds-base; // Formation  to USDWS conversion coeffient
+$u2f-coeff: $uswds-base / $formation-base; // USDWS to Formation conversion coeffient
```

Comment:
I got rid of sass:math and `math.div` because they don't work properly in vets-website, which still relies on node-sass. Node-sass doesn't handle `math.div` too well. This file now mirrors what we originally had in [here](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/override-function.scss) which doesn't use sass:math either.

Reply:
I'm getting some warnings locally about this. Is the plan to update this once Formation (node-sass) is deprecated in vets-website? 

<img width="917" alt="Screenshot 2024-10-01 at 5 06 12 PM" src="https://github.com/user-attachments/assets/815df77e-9d0f-4f77-a1ab-eeb5eb1fdfa8">